src/components/AnacondaPage.jsx

```diff
@@ -37,13 +37,15 @@ export const AnacondaPage = ({ children, isFormDisabled, setIsFormDisabled, step
         }
     }, [isFormDisabled]);
 
+    const titleIsEmpty = title?.type?.() === null || !title;
```

Comment:
How is that different from just `!title`? `!null === true`

Reply:
title can be a null React.component as of https://github.com/rhinstaller/anaconda-webui/pull/323/commits/d7b86c791d9d84ebbd60cd5038ec01345baf25a1#diff-7768fc0c14b2131f2f8fe93ac2e875fb66357a0766198d965f8f5f35a308167bR202

But actually I will adjust and allow title to be only string. Reposting.

Reply:
Yes, I'm asking why you introduced the `title?.type?.() === null` check. How is that different from `!title`?

Reply:
Because a React.component that returns null is not === null, it's still a React.component so some Object type, so !title will be truthy :)
Moved the null check out, see repush.

Reply:
Hmm, but wouldn't that be more like `title?.().type === null`? (That still doesn't cover calling a render function or something, just a functional component). Also, that would be `title?() === null`. 

To me this is absolutely incomprehensible, and it should get a comment what exactly happens here. Anyway, it's moot now as you dropped it.